src/watchmaker/workers/salt.py

```diff
@@ -452,16 +465,16 @@ def process_grains(self):
         self._set_grain('watchmaker', ent_env)
 
         grain = {}
-        if self.ou_path and self.ou_path != 'None':
+        if self.ou_path:
             grain['oupath'] = self.ou_path
-        if self.admin_groups and self.admin_groups != 'None':
+        if self.admin_groups and hasattr(self.admin_groups, "split"):
```

Comment:
i don't think we need the suspenders here. `self.admin_groups` should be a string by this point or `None`

Reply:
Done